File: Calculator/src/CalculatorBundle/Entity/Calculator.php
<?php
namespace CalculatorBundle\Entity;

class Calculator{
    /**
     * @var float
     */
    private $leftOperand;
    /**
     * @var float
     */
    private $rightOperand;
    /**
     * @var string
     */
    private $operator;
    /**
     * Get left operand
     *
     * @ retutn float
     */
    public function getLeftOperand(){
        return $this->leftOperand;
    }
    /**
     * Set left operand
     *
     * @param float $operand
     *
     * @return Calculator
     */
    public function setLeftOperand($operand){
        $this->leftOperand = $operand;
        return $this;
    }
    /**
     * Get right operand
     *
     * @ retutn float
     */
    public function getRightOperand(){
        return $this->rightOperand;
    }
    /**
     * Set right operand
     *
     * @param float $operand
     *
     * @return Calculator
     */
    public function setRightOperand($operand){
        $this->rightOperand = $operand;
        return $this;
    }
    /**
     * Get operator
     *
     * @ retutn string
     */
    public function getOperator(){
        return $this->operator;
    }
    /**
     * Set operator
     *
     * @param string $operand
     *
     * @return Calculator
     */
    public function setOperator($operand){
        $this->operator = $operand;
        return $this;
    }
}

?>

File: JavaSyntaxBasicWebCalculatorSkeleton/src/main/java/com/softuni/entity/Calculator.java
package com.softuni.entity;

public class Calculator {
    private double leftOperand;
    private double rightOperand;
    private String operator;

    public double getLeftOperand() {
        return leftOperand;
    }

    public void setLeftOperand(double leftOperand) {
        this.leftOperand = leftOperand;
    }

    public double getRightOperand() {
        return rightOperand;
    }

    public void setRightOperand(double rightOperand) {
        this.rightOperand = rightOperand;
    }

    public String getOperator() {
        return operator;
    }

    public void setOperator(String operator) {
        this.operator = operator;
    }

    public double calculateResult(){
        double result;
        switch (this.operator){
            case "+":
                result=this.getLeftOperand()+this.getRightOperand();break;
            case "-":
                result=this.getLeftOperand()-this.getLeftOperand();break;
            case "*":
                result=this.getLeftOperand()*this.getRightOperand();break;
            case "/":
                result=this.getLeftOperand()/this.getRightOperand();break;
            default:
                result=0;break;
        }

        return result;
    }
}
